app/timetables/admin.py

```diff
@@ -1,7 +1,7 @@
 from django.contrib import admin
 
 from .models import (Weekday, MealOption, Course, Meal, Timetable,
```

Comment:
@andela-eisaac I think explicitly importing them one by one would look good when referring to them in the code, as opposed to having `model.abc` everywhere.

For instance, in the admin.py file comparing the following
``` 
from . import models

 admin.site.register[models.abc, models.def, models.blablabla]
```

```
from .models import abc, def, blablabla

 admin.site.register[abc, def, blablabla]
```
I would prefer the second instead of the first.